quantum/quantum.h

```diff
@@ -38,11 +38,11 @@
 #endif
 
 #if defined(RGBLIGHT_ENABLE)
-#    include "rgblight.h"
+#    include "rgblight\rgblight.h"
 #elif defined(RGB_MATRIX_ENABLE)
 // Dummy define RGBLIGHT_MODE_xxxx
 #    define RGBLIGHT_H_DUMMY_DEFINE
-#    include "rgblight.h"
+#    include "rgblight\rgblight.h"
```

Comment:
> 
> 
> @drashna #13312 is already merged, AFAICT

#13312 is the change that broke compilation, the hack to fix it is actually #13371, which has not been merged.

The fix here and in `rgb_matrix.h` I suppose is equivalent and would probably also work, but there's presumably a reason why #13371 was done the way it was.

Reply:
IMHO using backslash for path separation is generally a bad idea. It just risks character escaping issues and problems on UNIX systems without any upside. Windows is the only thing that uses that as path separator *and* it also accepts forward slashes.

Probably both changes would be bettee go in and I have a hard time thinking backslashes here are actually required and/beneficial in any way. More often than not that's just people developing on Windows and doing Windows-y things in my experience.

Reply:
Or I might've misunderstood here.

The backslashes were added here in this PR, it seems? I thought they had come from somewhere else.

Yes, adding it to the VPATH is the correct fix. My comment on backslashes still stands in a way though: people developing on Windows and doing Windows-y things :D

Reply:
> 
> 
> Or I might've misunderstood here.
> 
> The backslashes were added here in this PR, it seems? I thought they had come from somewhere else.
> 
> Yes, adding it to the VPATH is the correct fix. My comment on backslashes still stands in a way though: people developing on Windows and doing Windows-y things :D

@andrebrait, the Windows-y path separator point was corrected in the following commit.

Reply:
@glorious-qmk yeah, I saw that :-)

Anyway, we need to wait for the other fix to go in and this has to be reverted after that, I guess